cf-deployment.yml

```diff
@@ -1526,6 +1521,27 @@ instance_groups:
         client_secret: ((uaa_clients_doppler_secret))
         internal_addr: https://uaa.service.cf.internal:8443
     release: log-cache
+- name: doppler
+  azs:
+  - z1
+  - z2
+  instances: 4
```

Comment:
Could we reduce the number of doppler instances to match the gain in log-cache instances (to some extent)? Is that a bad idea?

Reply:
Yes, let's reduce to 3 so log-cache vms + doppler vms still equals 4.